reporters_db/data/reporters.json

```diff
@@ -8552,6 +8608,28 @@
             }
         }
     ],
+    "Haz. Reg. Pa.": [
+        {
+            "cite_type": "scotus_early",
+            "editions": {
+                "Haz. Reg. Pa.": {
+                    "end": "1836-12-31T00:00:00",
+                    "regexes": [
+                        "$full_cite",
+                        "$volume $reporter \\(\\d{4}\\) $page"
```

Comment:
this looks wrong.   what is going on here?  
we have a regex pattern for years no?

Reply:
I see you followed a pattern started on another reporter.

I think what we should do is 

```
"year_included": {
            "": "$volume $reporter \\((?P<year>1[789]\\d{2}|20\\d{2})\\) $page",
            "#": "Citation that includes the volume year after the reporter '14 Haz. Reg. Pa. (1834) 10'"
        },
```

add something like this to the regex pattern 

and then we can just do 

```
"regexes": [
     "$full_cite",
     "$full_cite_year_included"
],
```

@mlissner @jcushman  it adds the year nicely into eyecite object, but having done just a few regexes I don't know what others think.

Reply:
> this looks wrong. what is going on here? we have a regex pattern for years no?

In fact, that is what I did. But you are absolutely right, it would be better to have this as a unique regex to avoid repeating regexes.

Reply:
I implemented this regex in this PR: https://github.com/freelawproject/reporters-db/pull/113